src/com/goide/inspections/GoUnusedImportDeclaration.java

```diff
@@ -70,6 +70,9 @@ protected void checkFile(@NotNull GoFile file, @NotNull ProblemsHolder problemsH
     }
 
     for (Map.Entry<String, Collection<GoImportSpec>> specs : importMap.entrySet()) {
+      if (".".equals(specs.getKey()) || "_".equals(specs.getKey())) {
```

Comment:
Please reimplement with invoking isDot and isForSideEffects on imports.next() object